components/raftstore-v2/src/operation/command/admin/compact_log.rs

```diff
@@ -487,12 +507,13 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             {
                 error!(self.logger, "failed to compact raft logs"; "err" => ?e);
             }
+            self.compact_log_context_mut().set_last_compacted_idx(index);
             // Extra write set right above.
         }
 
         let context = self.compact_log_context_mut();
         let applied = context.last_applying_index;
-        let total_cnt = applied - old_truncated;
+        let total_cnt = applied - prev_first_index;
```

Comment:
let total_cnt = applied - prev_first_index + 1; ?

Reply:
I think without `+1` makes it identical to with raftstore v1, see `PeerFsmDelegate::on_ready_compact_log`.